addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -1959,31 +2096,38 @@ export class OdooEditor extends EventTarget {
     // TOOLBAR
     // =======
 
-    toolbarHide() {
-        this._updateToolbar(false);
+    toolbarHide(force=false) {
+        this._updateToolbar(false, force);
     }
-    toolbarShow() {
-        this._updateToolbar(true);
+    toolbarShow(force=false) {
+        this._updateToolbar(true, force);
     }
     /**
      * @private
      * @param {boolean} [show]
+     * @param {boolean} [force=false] set true to force the value of `show`,
+     *                                ignoring other checks
      */
-    _updateToolbar(show) {
+    _updateToolbar(show, force=false) {
```

Comment:
That's one of the annoying things with having two different ways to define a selection. The way it currently is, whether you pass `show === true` or not to `_updateToolbar`, the toolbar can say no because it checks the selection first to see if it should be displayed or not. But these checks are irrelevant if we have a table selection because it's not the same selection...